### PR TITLE
Makes #87029 Opt-Out

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Adds a new boolean that lets you opt out of automatically linking zlevels in `LoadGroup()` (as implemented in https://github.com/tgstation/tgstation/pull/87029); and forwards that support to map jsons. Does ***not*** change any /tg/ map jsons in the process; as they all still work as intended.

A modified Icebox JSON loading correctly:
image

### How do I disable automatic z-level linking?
Add the following to your json; under planetary:
```	"height_autosetup": 0, ```

For each z-level's trait; you can then add either `"Up": true` or  `"Down": true` to link them manually, as you would've prior to #87029 .
## Why It's Good For The Game
tl;dr, map experimentation. It was prior possible to have station segments separated by z-level that weren't vertically linked; which was put to practice most prominently on Nova Sector, of all places, - but #87029 made it automatically assume that they should be linked in that manner.

This middleground keeps the new assumption while preserving the old manual process for any mapping projects that need it.
## Changelog
:cl:
code: Mappers can now opt out of automatically linking their up/down station traits.
/:cl:

### CODE REVIEW

**Review by san7890 — APPROVED** [review]
sounds fine to me
